pkg/odo/cli/component/push.go

```diff
@@ -35,7 +37,7 @@ type PushOptions struct {
 	*CommonPushOptions
 
 	// devfile path
-	devfilePath string
+	DevfilePath string
```

Comment:
why was this converted to a public variable?

Reply:
Because if there is a `now` flag provided, we need to do run `devfilePush`, which requires existance of a devfilePath in the `PushOptions`.

Reply:
OK